regression-tests/run-tests.sh

```diff
@@ -32,7 +32,7 @@ check_file () {
         # Add the file to the index to be able to diff it...
         git add "$file"
         # ... print the diff ...
-        git --no-pager diff HEAD -- "$file"
+        git --no-pager diff HEAD -- "$file" | tee -a "$cxx_compiler-patch.diff"
```

Comment:
Should this output be added at line 45 as well?
Unrelated, typo in "reference" at line 41.

Reply:
Probably.

Reply:
Yes, this should also be done in line 45.